src/components/images/ImageList.js

```diff
@@ -1,44 +1,72 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { fetchPhoto } from '../../actions';
+import { getRecent } from '../../actions';
 
+import SearchBar from '../SearchBar';
 import ImageDetail from './ImageDetail';
 
 class ImageList extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = { searchTerm: null };
+    }
 
     componentDidMount() {
-        this.props.fetchPhoto();
+        this.props.getRecent();
     }
 
     renderList() {
         return this.props.photos.map(photo => {
             return (
-                <div className="item" key={photo.id} >
-                    <ImageDetail 
-                        farmId={photo.farm} 
-                        serverId={photo.server} 
-                        id={photo.id} 
-                        secret={photo.secret} />
-                </div>
+                <ImageDetail 
+                    key={photo.id}
+                    farmId={photo.farm} 
+                    serverId={photo.server} 
+                    id={photo.id} 
+                    secret={photo.secret} 
+                    title={photo.title}/>
             );
         })
     }
 
+    onSearch = (searchTerm) => {
+        this.setState({ searchTerm });
```

Comment:
Again, lets limit the amount of component state we use. Redux is a cleaner way on storing state.